examples/remotivelabs-feeder/README.md

```diff
@@ -88,22 +96,28 @@ For a standard build of the playground with an unchanged IoTDB server configurat
 
 More typically you would change `device_id_` if needed to reflect how you are organizing data in IoTDB.
 
+For sending data to the information-layer the websocket server has to listen on `ws://localhost:8080`.
```

Comment:
~~Is the port and URL a default or a hard-coded value? 'Has to' suggests must.~~

Looking at the bridge source this is a hard-coded value. As there is no cmd line param to change it, I would explicitly make this a configuration item in the config section as I did for IoTDB.

Reply:
@chrizmc its getting on for three months since this comment. The PR is now a large code base that touches many parts of the Playground, so not getting it merged causes some development speed friction as you are either rebasing this a lot or related code on it. With that in mind something like this perhaps fits into the category of improvement backlog, rather than a gate blocking merge. The feeder could do with better runtime configuration anyway for our additions, so that could be done at the same time.

There are others here likely in the same category. We should discuss/categorize so we can efficiently both move towards the merge and know what lies on the other side in terms of back log.

Reply:
Pick this up #post-merge